pootle/apps/pootle_format/default.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+POOTLE_FORMATS = [
+    ("po",
+     dict(title='Gettext PO file',
+          extension="po",
+          template_extensions=["po", "pot"])),
```

Comment:
tbh it would greatly simplify things @dwaynebailey if we didnt need to support multiple template_extensions for a particular filetype